go-controller/pkg/ovn/master.go

```diff
@@ -782,6 +798,8 @@ func (oc *DefaultNetworkController) deleteNodeEvent(node *kapi.Node) error {
 			return err
 		}
 	}
+	oc.localZoneNodes.Delete(node.Name)
```

Comment:
it may be better to move this to line 806 below, so we ony do this after `oc.deleteNode` succeeded?